Mods/vcmi/mod.json

```diff
@@ -5,7 +5,7 @@
 	"chinese" : {
 		"name" : "VCMI essential files",
 		"description" : "Essential files required for VCMI to run correctly",
-		
```

Comment:
understood this is auto-formatting, but maybe let's exclude them from PR

Reply:
It just removes trailing spaces, I can convert it. Could you please tell me reason why we keep trailing spaces on blank lines?

Reply:
I think the idea was to not apply any autoformatting on things which do not relate directly to your PR. While these changes are small and it is ease to review them in general such approach can cause a lot of extra changes.

Reply:
Still I do not see any issues with removing spaces so think we can accept it as is.